app/models/event.py

```diff
@@ -29,20 +30,25 @@ def __str__(self):
 
     @property
     def noProgram(self):
-        return not self.programEvents.exists()
+        #Ask Anderson 
+        return not self.program_id.exists()
```

Comment:
you don't need .exists(), because `self.program` isn't a query, it's a value. Just do `not self.program`